Python/ast.c

```diff
@@ -3013,7 +3013,9 @@ ast_for_call(struct compiling *c, const node *n, expr_ty func,
                 e = ast_for_expr(c, CHILD(ch, 1));
                 if (!e)
                     return NULL;
-                kw = keyword(NULL, e, c->c_arena);
+                kw = keyword(NULL, e, chch->n_lineno, chch->n_col_offset,
+                             chch->n_end_lineno, chch->n_end_col_offset, c->c_arena);
+
```

Comment:
something like this can help
```suggestion
                kw = keyword(NULL, e, chch->n_lineno, chch->n_col_offset,
                             e->end_lineno, e->end_col_offset, c->c_arena);
```